src/Microsoft.ApiDesignGuidelines.Analyzers/Core/IdentifiersShouldNotContainTypeNames.cs

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.ApiDesignGuidelines.Analyzers
+{
+    /// <summary> 
+    /// CA1720-redefined: Identifiers should not contain type names 
```

Comment:
So "redefined" means, instead of warning on externally visible parameter/member names, we now warn regardless of visibility?